web-library/BookReservation/Entity/Reservation.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace web_library.BookReservation.Entity
+{
+    using System.ComponentModel.DataAnnotations;
+    using web_library.Book.Entity;
+
+    [Table("reservations")]
+    public class Reservation
+    {
+        [Column("id")]
+        [Key]
+        public int Id { get; set; }
+        [Column("reservation_start_date")]
+        public DateOnly reservationStartDate { get; }
+        [Column("reservation_end_date")]
+        public DateOnly reservationEndDate { get; }
```

Comment:
rezerwacja nie ma end date(zrob serwis ktory na podstawie daty sprawdza czy nie powinno sie jej zdjąć, jak jest wypozyczona to czekaj z wygasnieciem do konca wypozyczenia +3 dni, jak nie to poprostu 3 dni